stable/cluster-autoscaler/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 0.12.3
+version: 0.12.4
```

Minor version bump since this is new functionality.

```suggestion
version: 0.13.0
```